modules/installation-bare-metal-agent-installer-config-yaml.adoc

```diff
@@ -76,6 +76,35 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 <9> The cluster network provider Container Network Interface (CNI) plug-in to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN` .
 <10> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
 <11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
++
+[NOTE]
+====
+When you set the platform to `vSphere` or `baremetal`, you can configure IPv4 and IPv6 address endpoints for cluster nodes.
```

Comment:
I suggest making it clear that there 3 ways that IP addresses can be set up - either IPv4, IPv6, or both IPv4 and IPv6 i.e. DualStack, and that the example below shows DualStack.

Reply:
Implemented